### PR TITLE
feat(footer): Create a fully responsive footer with copyright symbol and app version

### LINKED ISSUE #75: [FEATURE] Create a footer
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

Create a footer

### Proposed Solution

Create the footer with copyright and the current version of the website.

### Screenshots

_No response_

### Do you want to work on this issue?

No

### If "yes" to above, please explain how you would technically implement this (issue will not be assigned if this is skipped)

_No response_

### PR DESCRIPTION
**Pull request by Piyush-Tilokani**
## Description
Closes #75 
Added a fully responsive footer with copyright and app version as requested in the issue.

Fixes #(issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots (if applicable):
image

image

@Bashamega please review and merge

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+const Navbar = () => {
+  return (
+    <div>
+      <footer className="footer footer-center bg-base-300 text-base-content p-10">
+        <aside>
+          <p>
+            Copyright © {new Date().getFullYear()} - All right reserved by
+            TrendTrack - v0.17.0
```
Comment (Bashamega) [inline src/components/Footer.tsx:9]: Make the verson automatic. You can get it from the package.json file

**Review by Bashamega — CHANGES_REQUESTED** [review]

**Comment by Bashamega** [comment]
Hello @Piyush-Tilokani I have reviewed your pr. Please do not ping me, I will check any notifications i have in less than 24 hours.

**Comment by Piyush-Tilokani** [comment]
Updated the code in the latest commit to fetch app version dynamically from package.json

**Review by Bashamega — APPROVED** [review]
Nice, thanks for contributing.

**Pull request merged**